### DISCUSSION TITLE
Welcome to OpenFE Discussions!

### DISCUSSION THREAD

**Discussion by ZhangTP1996** [IIIS-Li-Group/OpenFE#7] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

**Comment by avillasmil** [comment]
Hello, I was wondering how I could go about determining the bin edges that are computed for freq(x) features? Where are those stored within the object? Thanks!
